x-pack/plugins/secops/common/graphql/types.ts

```diff
@@ -37,10 +37,12 @@ export interface Query {
 export interface Source {
   id: string /** The id of the source */;
   configuration: SourceConfiguration /** The raw configuration of the source */;
+  getSuricataEvents?: (SuricataEvents | null)[] | null /** Get all event from suricata */;
```

Comment:
Would it be possible to always accept non-`null` input, and always return an empty array when none of the data matches the search criteria?

Reply:
```
/** Get all event from suricata */
```

`nit:` does this function get _all_ events, or does it retrieve events based on criteria?

Reply:
it can do both

Reply:
What do you think of a description like the following?:

"Gets Suricata events based on the specified criteria, or all events if no criteria is specified"

Reply:
👍